docs/illustrations/Basics.md

```diff
@@ -0,0 +1,50 @@
+---
+stoplight-id: xndc5l7aksqmw
+---
+
+# Basics
+
+## What is an illustration?
+
+An illustration is a document which models a product over time. They are predominantly used for pension products, which must be produced according to regulation. 
+
+An accumulation illustration models the growth of a product based on contributions or transfers added to the product, expected investment growth and the impact of fees. Growth rates are governed by legislation, whilst fees should represent those which are applicable to the customer. Figures are an estimation based on the details provided and the value of the product could be more or less than what has been projected.
+
+Drawdown illustrations are similar to accumulation illustrations, but these cover at-retirement options where funds can be taken from a pension. Investment growth and fees work in the same way compared to an accumulation illustration, however income (instead of contributions) may be modelled in the illustration. 
```

Comment:
If we're going to give information about drawdown illustrations here, I think it's worth mentioning that we don't support them at the end of this paragraph (as well as in the FAQs, as you've already done)

Reply:
Added a new section to show the supported scope and pulled the accumulation inputs section up into this